src/modules/fancyzones/editor/FancyZonesEditor/CanvasZone.xaml.cs

```diff
@@ -197,8 +210,9 @@ public override void Move(int delta)
         private SnappyHelperBase snappyX;
         private SnappyHelperBase snappyY;
 
-        private SnappyHelperBase NewDefaultSnappyHelper(bool isX, ResizeMode mode, int screenAxisSize)
+        private SnappyHelperBase NewDefaultSnappyHelper(bool isX, ResizeMode mode)
```

Comment:
`NewDefaultSnappyHelper` wasn't added by this PR, but the name doesn't sound right.

Based on this comment
```
            /// Initializes a new instance of the <see cref="SnappyHelperBase"/> class.
            /// Just pass it the canvas arguments. Use mode
            /// to tell it which edges of the existing masks to use when building its list
            /// of snap points, and generally which edges to track. There will be two
            /// SnappyHelpers, one for X-coordinates and one for
            /// Y-coordinates, they work independently but share the same logic.
```
 Do we actually have two `SnappyHelpers` for the X and Y coordinates?
I see `NewDefaultSnappyHelper` and `SnappyHelperMagnetic`
Can we refactor the name `NewDefaultSnappyHelper` to be more meaningful?
And also update the cited comment?

Reply:
> Feature works nicely only if monitors have same scaling. Is that what was intended? It would make more sense to work if monitors have same (resolution x scaling)

Yes, for now, we'll only support the same scaling on all monitors.

Reply:
Wait, how did my reply get here?

Reply:
We can do this in another PR